src/middleware/admin/checkStatus.js

```diff
@@ -0,0 +1,23 @@
+const Joi = require ('joi')
+
+const checkStatus = Joi.object().keys({
+    status: Joi.string().
+    valid('pending', 'in-progress', 'completed', 'rejected').trim().required(),
+  
+});
+
+const ChangeStatusMiddleware = (request, response, next) => {
```

Comment:
you naming convention here is wrong. Variables starting with capital letters are meant for classes, not regular functions.

Change this to `changeStatusMiddleware`